scripts/provisioning/s3_start

```diff
@@ -40,6 +40,8 @@ def main():
 
   parser = argparse.ArgumentParser("S3server single entry command")
   parser.add_argument("--service", required=True)
+  parser.add_argument("--instance", required=False)
+  parser.add_argument("--s3config", required=False)
```

Comment:
Add --config as generic thing .
--instance = s3 will implicitely require s3 config to be passed

Also - make provision for authserver, bgprod and bgconsumer as well as service

Reply:
@nileshgovande -- this PR is specific for s3server.  auth is already added. BG are separate JIRAs and separate PRs

discussed with Abhilekh -- s3config path will be hard coded for now, and updated through conf store whenever keys are added and ready for use.